tools/docker-images/clp-package/build.sh

```diff
@@ -4,11 +4,26 @@ set -eu
 set -o pipefail
 
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+iid_file="${script_dir}/image.id"
 repo_root=${script_dir}/../../../
 
+# Remove the previous image after the build to allow layer reuse.
```

Comment:
maybe i should phrase it as "image export reuse"?

like delaying the image delete saves 6s in the image export step on my computer